apps/dashboard/app/assets/stylesheets/breadcrumb.scss

```diff
@@ -10,3 +10,18 @@
 .breadcrumb-no-delimiter .breadcrumb-item + .breadcrumb-item::before {
   padding-right: 0;
 }
+
+.breadcrumb-wrapper {
+  --breadcrumb-bg: #e9ecef;
+  --breadcrumb-margin-bottom: 1rem;
```

Comment:
Not sure if I see the value in having variables here if they're only used once. Can you tell me why we wouldn't just use the constants below?